releasenotes/notes/prelude-release-7.52.1-55ab00a2138cb7dc.yaml

```diff
@@ -0,0 +1,5 @@
+prelude:
+    |
+    Release on: 2024-04-03
+
+    - Please refer to the `7.52.1 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7521>`_ for the list of changes on the Core Checks
```

Comment:
```suggestion
    - Refer to the `7.52.1 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7521>`_ for the list of changes on the core checks.
```

Reply:
@kacper-murzyn  Oh okay, if it's been like that then let's just leave it for now.